peachjam/js/components/DocumentSearch/HTMLSnippets.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div
+    v-for="(snippet, index) in snippets"
+    :key="index"
+    class="card mb-2"
+  >
+    <div class="card-body">
+      <ResultSnippet
+        class="mb-2"
+        :node="snippet.cloneNode(true)"
+      />
+      <div>
+        <a
+          href="#"
+          @click.prevent="$emit('go-to-snippet', snippet);"
+        >
+          Go to result
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import ResultSnippet from './ResultSnippet.vue';
+export default {
+  name: 'HTMLSnippets',
+  components: {
+    ResultSnippet
+  },
+  props: {
+    nodes: {
+      type: Array,
+      required: true
+    }
+  },
+  emits: ['go-to-snippet'],
+  data: () => ({
+    snippets: []
+  }),
+  watch: {
+    nodes: {
+      deep: true,
```

Comment:
I just want to check if you really need `deep: true`? PDFSnippets doesn't have it, and I suspect that the array is always reset by the parent component, rather than attributes on the actual nodes changing.

Reply:
you're right, we dont actually need to watch deep for this. refactored the watchers